### PR TITLE
Add CLI option to generate a cert for timestamping

### PR DESCRIPTION
**Pull request by asraa**
Let's users also create a cert issued for timestamping through the CLI. Built on top of:
https://github.com/sigstore/fulcio/pull/77

Co-authored-by: Priya Wadhwa <priyawadhwa@google.com>
Signed-off-by: Asra Ali <asraa@google.com>

### CODE REVIEW

**Comment by lukehinds** [comment]
We should try to move this forward. Is this needed for the offline signature bundles. This could really use a linked issue to describe what work this is part of.

File: cmd/client/app/generate.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
+/*
+Copyright © 2021 Dan Lorenc <lorenc.d@gmail.com>
```
Comment (hectorj2f) [inline cmd/client/app/generate.go:2]: Shouldn't this be `Copyright 2021 The Sigstore Authors.` ?

Comment (hectorj2f) [reply cmd/client/app/generate.go:2]: Oh this is an old PR, nvm

**Comment by asraa** [comment]
@hectorj2f @lukehinds Right now I can close it, we shouldn't need it except to generate certs by hand for rekor's keys. End users shouldn't need this

**Pull request closed without merging**